### PR TITLE
Change page title to "Avocado Testing Framework"

### PR DESCRIPTION
**Pull request by adereis**
It was "Avocado - Testing made easier", which IMO is
not the best slogan for the kind of framework we provide
today.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <meta name="description" content="Avocado - Automated Testing Framework">
     <meta name="author" content="">
 
-    <title>Avocado - Testing made easier</title>
+    <title>Avocado Testing Framework</title>
```
Comment (ldoktor) [inline index.html]: Yep, this should even simplify the searching.

Comment (ldoktor) [inline index.html]: Actually this only changes the window name. How about changing the main heading on line 71? Also you should probably adjust the meta description (line 9) in order to keep it in sync.

Comment (adereis) [inline index.html]: I don't want to keep them all in sync. I think it's positive to have "Avocado Testing Framework", "Avocado - Automated Testing Framework" and "Avocado Framework" all there. It doesn't bother me at all and I don't think it hurts.

What bothers me is the headline "Testing Made Easier", which is the only thing I'm changing here.

Comment (ldoktor) [inline index.html]: OK, I don't have any strong opinion, but the window title `Avocado Framework` looks indeed better. Merging...

**Comment by lmr** [comment]
Sure.

**Pull request merged**